emcc.py

```diff
@@ -806,7 +801,7 @@ def setting_sub(s):
 
         if not arg.startswith('-'):
           if not os.path.exists(arg):
-            exit_with_error('%s: No such file or directory ("%s" was expected to be an input file, based on the commandline arguments provided)' % (arg, arg))
```

Comment:
Not sure if I overall like deferring the %-format. I do like how it's a bit syntactically lighter, but I don't like how it isn't obvious from the callsite that these are format-string params.